index.html

```diff
@@ -167,6 +167,20 @@ <h4 class="feature-title mt-24">Receive back BTC</h4>
                 </div>
             </section>
 
+            <section class="pricing section">
+                <div class="pricing-header text-center">
+                    <h2 class="section-title mt-0">Demo version live on Bitcoin testnet</h2>
+                    <p class="section-paragraph mb-0"></p>
+                    <div class="hero-cta"><a class="button button-primary" href="https://itchysats.medium.com/p2p-bitcoin-cfds-give-it-a-try-4db2d5804328" target="_blank">Try it yourself!</a></div>
+                </div>
+                <div class="container">
+                    <video width="100%" controls autoplay>
+                        <source src="dist/images/screen_record_demo.mov" type="video/mp4">
```

Comment:
Can we use a youtube link instead? 

The problem with an embedded video is that it will slows down the website

Reply:
Can do, but I find the youtube embedded iframe ugly end distracting. I think with a modern browser using HTML5 videos should not be an issue, but in the end I don't mind much. 
We will need some CSS magic to scale the iframe properly I found some and slapped it in...

Reply:
![image](https://user-images.githubusercontent.com/88706813/141969826-6e8421bb-be1c-4b79-8989-f5dcdb99513a.png)

Reply:
I think that looks good :) well done.